### PR TITLE
Landing page validation + test

### PR DESCRIPTION
**Pull request by wg102**
Related to https://jirab.statcan.ca/browse/BTIS-221

### CODE REVIEW

File: components/centraldashboard/public/components/landing-page.js [hunk 1]
```diff
@@ -106,7 +108,7 @@ export class LandingPage extends mixinBehaviors([AppLocalizeBehavior], utilities
     }
 
     async logout() {
-        return this.buildHref(`/logout`);
+        location.href = `/logout`;
```
Comment (mathis-marcotte) [inline components/centraldashboard/public/components/landing-page.js:111]: What happened to the buildHref function? Did it get deleted by mistake? This is the same cause of why the buttons in the default notebook card aren't working. Might be a better idea to just add that back if it's causing issues in multiple places.
Although, now that I think about about it, that may have been a function in main-page.js, so I don't think calling it with "this" would work. So did this logout function in the landing-page.js file ever even worked?

**Review by mathis-marcotte — CHANGES_REQUESTED** [review]

File: components/centraldashboard/public/components/landing-page.js (hunk 1 above)
Comment (wg102) [reply components/centraldashboard/public/components/landing-page.js:111]: No the logout did not work. I tried to use the build href but it did not behave, so I looked up stackoverflow and this was the suggested approach.

Comment (mathis-marcotte) [reply components/centraldashboard/public/components/landing-page.js:111]: Didnt behave because it didn't exist. But yeah location.href should work.

**Review by mathis-marcotte — APPROVED** [review]
lgtm

**Pull request merged**